lib/generators/suspenders/ci_generator.rb

```diff
@@ -13,7 +13,7 @@ class CiGenerator < Rails::Generators::Base
 
       def ci_files
         empty_directory ".github/workflows"
-        template "ci.yml", ".github/workflows/ci.yml"
+        template "ci.yml", ".github/workflows/ci.yml", force: true
```

Comment:
Is it possible/worth adding test coverage for this by first adding a dummy `.github/workflows/ci.yml` in the setup, and ensuring our template overrides it?

Reply:
Oh, yes! That'd do it (and make me happy.)

Reply:
This did indeed work well, thanks for the suggestion!